frontend/src/tests/components/OurTable.test.js

```diff
@@ -1,674 +1,432 @@
-        fireEvent.click(nextButton);
-        expect(await screen.findByTestId("testid-current-page-button")).toContainHTML("5");
-        expect(await screen.findByTestId("testid-first-page-button")).toBeInTheDocument();
-        const firstButton = screen.getByTestId("testid-first-page-button");
-        fireEvent.click(firstButton);
-        expect(await screen.findByTestId("testid-current-page-button")).toContainHTML("1");
-    });
-
-    test("renders a table with 100 rows and tests the back two page button", async () => {
-        render(
-            <OurTable columns={columns} data={hundredRows} />
-        );
-
-        expect(await screen.findByTestId("testid-next-page-button")).toBeInTheDocument();
-        const nextButton = screen.getByTestId("testid-next-page-button");
-        fireEvent.click(nextButton);
-        expect(await screen.findByTestId("testid-current-page-button")).toContainHTML("2");
-        var tester = true;
-        try {
-            await screen.findByTestId("testid-back-two-page-button");
-            tester = false;
-        } catch(e) { }
-        expect(tester).toBe(true);
-        fireEvent.click(nextButton);
-        expect(await screen.findByTestId("testid-current-page-button")).toContainHTML("3");
-        expect(await screen.findByTestId("testid-back-two-page-button")).toBeInTheDocument();
-    });
-
-    test("renders a table with 100 rows and tests the last page button visibiliy", async () => {
-        render(
-            <OurTable columns={columns} data={hundredRows} />
-        );
-
-        expect(await screen.findByTestId("testid-last-page-button")).toContainHTML("10");
-        const lastButton = screen.getByTestId("testid-last-page-button");
-        fireEvent.click(lastButton);
-        expect(await screen.findByTestId("testid-current-page-button")).toContainHTML("10");
-        expect(await screen.findByTestId("testid-back-one-page-button")).toContainHTML("9");
-        const backOneButton = screen.getByTestId("testid-back-one-page-button");
-        fireEvent.click(backOneButton);
-        expect(await screen.findByTestId("testid-current-page-button")).toContainHTML("9");
-        expect(await screen.findByTestId("testid-forward-one-page-button")).toContainHTML("10");
-        expect(lastButton).not.toBeInTheDocument();
-        fireEvent.click(backOneButton);
-        expect(await screen.findByTestId("testid-current-page-button")).toContainHTML("8");
-        expect(lastButton).not.toBeInTheDocument();
-        fireEvent.click(backOneButton);
-        expect(await screen.findByTestId("testid-current-page-button")).toContainHTML("7");
-        expect(lastButton).not.toBeInTheDocument();
+  }
+
+  const fiftyOneRows = [];
+  for (let i = 1; i <= 51; i++) {
+    fiftyOneRows.push({
+      col1: `Hello ${i}`,
+      col2: `World ${i}`,
+      createdAt: `2021-04-01T04:00:00.000`,
+      log: `foo\nbar\n  baz ${i}`,
     });
+  }
+
+  const clickMeCallback = jest.fn();
+
+  const columns = [
```

Comment:
I think I have to disagree with this since seeing the jest-related stuff immediately prior to the tests themselves seems more readable than having the jest-related code, then the "fixtures" replacement, and then the tests.